src/main/java/org/isf/admission/service/AdmissionIoOperations.java

```diff
@@ -280,7 +280,7 @@ public static LocalDateTime getNow() {
 	 */
 	public boolean setDeleted(int admissionId) throws OHServiceException {
 		Admission foundAdmission = repository.findById(admissionId).orElse(null);
-		foundAdmission.setDeleted("Y");
+		foundAdmission.setActive(Integer.valueOf(1));
```

Comment:
here it should either set active to 0 or just use the delete method from JPARepository, I suppose